### ISSUE TITLE
Hypatia fails to precompile on 1.10

### ISSUE THREAD

**Issue by ranjanan** [chriscoey/Hypatia.jl#835]
```julia
ERROR: LoadError: UndefVarError: `SuiteSparse_long` not defined
Stacktrace:
 [1] getproperty(x::Module, f::Symbol)
   @ Base .\Base.jl:31
 [2] top-level scope
   @ D:\.julia\packages\Hypatia\lPACo\src\linearalgebra\sparse.jl:14
 [3] include(mod::Module, _path::String)
   @ Base .\Base.jl:495
 [4] include(x::String)
   @ Hypatia D:\.julia\packages\Hypatia\lPACo\src\Hypatia.jl:11
 [5] top-level scope
   @ D:\.julia\packages\Hypatia\lPACo\src\Hypatia.jl:20
 [6] include
   @ Base .\Base.jl:495 [inlined]
 [7] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::Nothing)
   @ Base .\loading.jl:2216
 [8] top-level scope
   @ stdin:3
in expression starting at D:\.julia\packages\Hypatia\lPACo\src\linearalgebra\sparse.jl:14
in expression starting at D:\.julia\packages\Hypatia\lPACo\src\Hypatia.jl:8
in expression starting at stdin:3
  ✗ Hypatia
  0 dependencies successfully precompiled in 2 seconds. 47 already precompiled.

ERROR: The following 1 direct dependency failed to precompile:

Hypatia [b99e6be6-89ff-11e8-14f8-45c827f4f8f2]

Failed to precompile Hypatia [b99e6be6-89ff-11e8-14f8-45c827f4f8f2] to "D:\\.julia\\compiled\\v1.10\\Hypatia\\jl_60C5.tmp".
ERROR: LoadError: UndefVarError: `SuiteSparse_long` not defined
Stacktrace:
 [1] getproperty(x::Module, f::Symbol)
   @ Base .\Base.jl:31
 [2] top-level scope
   @ D:\.julia\packages\Hypatia\lPACo\src\linearalgebra\sparse.jl:14
 [3] include(mod::Module, _path::String)
   @ Base .\Base.jl:495
 [4] include(x::String)
   @ Hypatia D:\.julia\packages\Hypatia\lPACo\src\Hypatia.jl:11
 [5] top-level scope
   @ D:\.julia\packages\Hypatia\lPACo\src\Hypatia.jl:20
 [6] include
   @ Base .\Base.jl:495 [inlined]
 [7] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::Nothing)
   @ Base .\loading.jl:2216
 [8] top-level scope
   @ stdin:3
in expression starting at D:\.julia\packages\Hypatia\lPACo\src\linearalgebra\sparse.jl:14
in expression starting at D:\.julia\packages\Hypatia\lPACo\src\Hypatia.jl:8
in expression starting at stdin:
```

**Comment by araujoms** [comment]
This has already been fixed with the PR #828, so you just need to switch to the development version of Hypatia to get it to compile.

**Comment by chriscoey** [comment]
Sorry for the delay here - I just tagged a new version, so you should be able to just update hypatia and use it on 1.10 now
